karma.conf.js

```diff
@@ -0,0 +1,82 @@
+// Karma configuration
```

Comment:
What is the essence of this file? Are you using Karma or Jasmine to run your test?